chat-commands.js

```diff
@@ -70,6 +70,16 @@ exports.commands = {
 		"/auth [room] - Show what roomauth a room has.",
 		"/auth [user] - Show what global and roomauth a user has."],
 
+	userlist: function (target, room, user) {
+		let userlist = room.getUserList().substr(7);
+		userlist = userlist.split(',').splice(1); // split userlist by comma, exclude usercount
```

Comment:
In hindsight, `getUserList` is probably wrong; implementing it yourself like you did before seems to be the best choice.